### DISCUSSION TITLE
USR-W610 in UDP, not TCP

### DISCUSSION THREAD

**Discussion by Chrisbrown01942** [nathanmarlor/foxess_em#295] [Q&A] [answered]
Hi, I have my USR-W610 working in UDP, not TCP. Configuring the Energy Management integration results in a connection error, but works fine in TCP. Am I right in guessing the integration will only work with the USR-W610 set to TCP?

Thanks, Chris

**Comment by nathanmarlor** [comment, marked as answer]
Ah yes, I haven't configured EM for UDP (yet).
